stack.yaml

```diff
@@ -1,16 +1,8 @@
-resolver: lts-9.11
+resolver: lts-11.7
 packages:
 - '.'
 extra-deps:
-- aeson-1.1.0.0
-- genvalidity-0.3.1.0
-- genvalidity-hspec-0.3.1.0
-- genvalidity-hspec-aeson-0.0.1.0
-- genvalidity-path-0.1.0.0
-- directory-1.3.0.2
+- HUnit-1.5.0.0
 - iostring-0.0.0.0
-- optparse-applicative-0.13.1.0
-- process-1.5.0.0
-- validity-0.3.2.0
-- validity-path-0.1.0.0
-resolver: lts-7.19
+- path-0.5.13
```

Comment:
I'd prefer to use newer paths. `>= 0.6` if we're changing versions anyway.

Reply:
The path bound can't be updated as long as the iostring bounds aren't updated.

Reply:
Get rid of iostring then. It's deprecated anyway.